src/fhir_mcp_server/utils.py

```
@@ -125,6 +129,76 @@ async def get_capability_statement(metadata_url: str) -> Dict[str, Any]:
+        if "resource" in data and isinstance(data["resource"], dict):
+            # raw bundle entry wrapper {"fullUrl": ..., "resource": {...}, "search": ...} — filter the resource inside
+            result = _apply_fhirpath(data["resource"], expressions)
+            if "search" in data:
+                result["search"] = data["search"]  # preserve match/include mode for _include queries
+            return result
+        # single resource
+        return _apply_fhirpath(data, expressions)
+    return data
+
+
+def format_response(data: Any, format: str = "toon") -> Any:
```

Ruff (0.15.12)

[error] Function argument `format` is shadowing a Python builtin

(A002)